heft-plugins/heft-lint-plugin/src/test/snapshot.test.ts

```diff
@@ -0,0 +1,44 @@
+// Copyright (c) Microsoft Corporation. All rights reserved. Licensed under the MIT license.
+// See LICENSE in the project root for license information.
+import { formatEslintResultsAsSARIF } from '../SarifFormatter';
+import type { ISerifFormatterOptions } from '../SarifFormatter';
+import type { ESLint } from 'eslint';
+
+describe('formatAsSARIF', () => {
+  test('should correctly format ESLint results into SARIF log', () => {
```

Comment:
Include some more varied cases. For example:

- A case with no files
- A case with multiple issues in the same file
- A case with a file with no messages
- A case with multiple files
- A case with `ignoreSuppressed: true`, and with and without suppressions
- A case without file locs

Reply:
Added snaptests for all cases above.